libs/data-mapper/src/lib/components/propertiesPane/tabs/FunctionNodePropertiesTab.tsx

```diff
@@ -202,16 +204,25 @@ export const FunctionNodePropertiesTab = ({ functionData }: FunctionNodeProperti
 
                 return (
                   <div key={index} style={{ marginTop: 8 }}>
-                    <Tooltip relationship="label" content={input.tooltip || ''}>
-                      <InputDropdown
-                        currentNode={functionData}
-                        placeholder={input.placeHolder}
-                        inputName={getInputName(inputConnection)}
-                        inputValue={getInputValue(inputConnection)}
-                        inputIndex={index}
-                        inputAllowsCustomValues={input.allowCustomInput}
-                      />
-                    </Tooltip>
+                    {input.displayAttribute === InputFormat.TextBox && (
+                      <InputTextbox
```

Comment:
Should we have an info bubble or something to put the tool tip into?

Reply:
https://github.com/Azure/LogicAppsUX/issues/3073 made a separate workitem for this!